docs/controls/WebView-known-issues.md

```diff
@@ -0,0 +1,112 @@
+
+## Performance
+
+* **WebView** controls in WPF applications have longer load times than Windows Forms applications.
+
+* If the web view hangs or stops working, other applications that use the same underlying process (Win32WebViewHost) will hang or stop working as well.
+
+## Security
+
+* **WebView** controls don't render in XAML Browser Applications (XBAP).
+
+  That's because a browser has partial trust to the system while WebView controls have full-trust.
+
+* **WebView** controls won't appear if the Win32WebViewHost application is disabled in the location settings of the user's system.
+
+* **WebView** controls won't function as expected in a WPF-based ClickOnce application.
+
+* **WebView** controls function only in applications that use a Single-Threaded Apartment (STA) model. Multithreaded Apartment (MTA) model is not supported.
+
+* **WebView** controls have not been tested for Windows Information Protection. Therefore, the information that moves in and out of the WebView control might not be properly secured even if an  organizational administrator applies a Windows Information Protection policy to the hosting application.  
+
+* **WebView** controls have not been exhaustively tested for security.
+
+## User interaction
+
+* Accelerator keys don't work in **WebView** controls.
+
+* Users can't use the TAB key to put the **WebView** into focus. They'll have to use the mouse or other pointing device to put the **WebView** control into focus.
+
+## Paths and strings
+
+* You can use only fully qualified paths to resources in members of the **WebView** control that accept string paths. **WebView** controls don't recognize the File:// or ms-appx:/// prefixes so they can't read from the file system or the package (if you've created a package for your application).
+
+* **WebView** controls don't support URI's that are not encoded in UTF-8. Characters between UTF-16 and UTF-32 aren't supported.
+
+* **WebView** controls don't support URIs that are greater than 2048 characters.
+
+## Host container
+
+* **WebView** controls can be hosted in a popup window. We recommend that you do not do this because support for that scenario will soon be disabled for security reasons.
+
+* Web applications that run in a **WebView** control can't initiate downloads.
+
+* Service workers can't run in a **WebView** control.
+
+* Your code can't instantiate more than one instance of a **WebView** in the same application.
+
+## WebView browser
+
+* The [WebBrower.ObjectForScripting](https://msdn.microsoft.com/library/system.windows.controls.webbrowser.objectforscripting.aspx) property is not supported.
+
+  Instead, use the [WebBrowser.InokeScript](https://msdn.microsoft.com/library/cc491132.aspx) method.
+
+* You can't programmatically navigate by using a new window.
+
+* You can't programmatically navigate to a specific frame.
+
+* You can't programmatically navigate to a relative URI.
```

Comment:
Using `IWebView.Source`, `IWebView.Navigate(Uri)`, or `IWebView.Navigate(String)`

Reply:
Hmm. Are you suggesting that the sentence be "You can't programmatically navigate to a relative URI by using the IWebView.Source property or the IWebView.Navigate(Uri) or IWebView.Navigate(String) methods? Why mention these? By doing so, it has my mind asking the question of - ok then which method can I use?  Seems that our message is that they can't do it at all right? Thanks

Reply:
You're right, I thought I saw it in the context of something else. I just think I've been looking at this too long